package.json

```diff
@@ -96,5 +96,5 @@
 	"engines": {
 		"node": ">=18"
 	},
-	"packageManager": "pnpm@8.10.2"
+	"packageManager": "pnpm@8.15.7+sha256.50783dd0fa303852de2dd1557cd4b9f07cb5b018154a6e76d0f40635d6cee019"
```

Comment:
What happened here?

Reply:
I also ran `corepack up` to bump pnpm to the latest (v8) version too